src/main/java/io/logz/logback/LogzioSender.java

```diff
@@ -25,8 +25,6 @@
     private final int INITIAL_WAIT_BEFORE_RETRY_MS = 2000;
     private final int MAX_RETRIES_ATTEMPTS = 3;
 
-    private final ScheduledThreadPoolExecutor tasksExecutor;
-
     private BigQueue logsBuffer;
```

Comment:
Let's have final for any class member which will not change (usually c'tor arguments). It's best practice.